phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -116,6 +111,7 @@ public void setOnSelectScript(Consumer<ScriptDTO> onSelectScript) {
      * Show available apps panel
      */
     public void showAvailableApps() {
+        this.showDetailsView(null);
```

Comment:
For this I would suggest adding a `clearDetailsView` method, because I think it's unintuitive to clear the view by setting it to `null`.